pennylane/qnn/keras.py

```diff
@@ -370,3 +370,14 @@ def input_arg(self):
         `Layer <https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer>`__. Set to
         ``"inputs"``."""
         return self._input_arg
+
+
+    def get_config(self):
```

Comment:
Even though this is an overwritten method, perhaps it might be good to include a short docstring here just so future developers understand why this is overwritten!